src/utils/request/fetchAllRequest.js

```diff
@@ -0,0 +1,12 @@
+import singleRequest from './singleRequest';
+
+const fetchAllRequest = async (url, amountOfRequests, fetchOptions = {}) => {
+  const data = await Promise.all(
+    [...new Array(amountOfRequests - 1)] // -1 because we already did a firstRequest
+      .map((_, i) => i + 2) // 2, 3, 4, ...
+      .map((_, number) => singleRequest(url, { ...fetchOptions, page: { number } })),
```

Comment:
🤔 This doesn't actually do anything as you are again using the index instead of the value here.

You're better off doing a single `.map` and adding `2` to the pagenumber in the same map

Reply:
So instead of

```js
.map((_, i) => i + 2) // 2, 3, 4, ...
.map((_, number) => singleRequest(url, { ...fetchOptions, page: { number } })),
```

go for simply

```
.map((_, i) => singleRequest(url, { ...fetchOptions, page: { i + 2 } })),
```

?

Reply:
Correct